praw/models/reddit/live.py

```diff
@@ -431,3 +431,9 @@ def remove(self):
         url = API_PATH['live_remove_update'].format(id=self.update.thread.id)
         data = {'id': self.update.fullname}
         self.update.thread._reddit.post(url, data=data)
+
+    def strike(self):
+        """Strike a content of a live update."""
```

Comment:
> Perhaps we could also indicate which attribute on an update indicates whether or not it has been stricken

It's not easy because `"kind": "LiveUpdate"` things have `stricken` attribute, but current implementation of `LiveUpdate` class sets `_fetched = True` inside of `__init__` due to the lack of proper API, so `update.stricken` immediately raises AttributeError when accessed.

Above limitation is annoying, so I'll send feature request to the upstream once prepared.